src/app/ubs-admin/components/ubs-admin-employee/employee-form/employee-form.component.ts

```diff
@@ -0,0 +1,76 @@
+import { Component, OnInit, Inject, Input } from '@angular/core';
+import { MatDialog, MAT_DIALOG_DATA } from '@angular/material/dialog';
+import { FormArray, FormBuilder, FormControl, FormGroup } from '@angular/forms';
+import { UbsAdminEmployeeService } from '../../../services/ubs-admin-employee.service';
+import { Validators } from '@angular/forms';
+
+export interface Position {
+  name: string;
+}
+@Component({
+  selector: 'app-employee-form',
+  templateUrl: './employee-form.component.html',
+  styleUrls: ['./employee-form.component.scss']
+})
+export class EmployeeFormComponent implements OnInit {
+  @Input()
+  locations;
+  roles;
+  selectedFile: File;
+  filePath: string;
+  uploaded = false;
+  employeeForm: FormGroup;
+  positionsArr = [];
+  ngOnInit() {
+    this.employeeService.getAllPositions().subscribe({
+      next: (roles) => {
+        this.roles = roles;
+      }
+    });
+    this.employeeService.getAllStantions().subscribe({
+      next: (locations) => {
+        this.locations = locations;
+      }
+    });
```

Comment:
instead of next let's be consistent and have this like subscribe(() => sucess, ()=> error)

Reply:
added error handling